includes/class-sensei-usage-tracking.php

```diff
@@ -113,12 +112,7 @@ public static function maybe_unschedule_tracking_task() {
 	 * @return bool
 	 **/
 	public static function is_tracking_enabled() {
-		$setting = Sensei()->settings->get( self::$usage_tracking_setting_name ) || false;
-		/**
-		 * Filter
-		 * @return bool
-		 **/
-		$enabled = (bool) apply_filters( 'sensei_usage_tracking_enabled', (bool)$setting );
+		$enabled = Sensei()->settings->get( self::$usage_tracking_setting_name ) || false;
```

Comment:
Could even turn this into a one-liner by returning the result directly if we were so inclined. No need for a variable really.

Reply:
Done!